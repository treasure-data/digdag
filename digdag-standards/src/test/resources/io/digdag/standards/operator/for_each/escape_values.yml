_command:
  u:
    - '!?'
    - '^$'
    - '%=='
    - '|~'
    - '[<{}>]'
    - '#:'
    - '/@'
    - '+ '
_do:
  echo>: value is ${u}
