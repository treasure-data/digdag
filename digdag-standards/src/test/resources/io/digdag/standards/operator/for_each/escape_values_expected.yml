"+for-0=u=0=%21%3F":
  echo>: value is ${u}
  _export: {u: "!?"}
"+for-0=u=1=%5E%24":
  echo>: value is ${u}
  _export: {u: "^$"}
"+for-0=u=2=%25%3D%3D":
  echo>: value is ${u}
  _export: {u: "%=="}
"+for-0=u=3=%7C%7E":
  echo>: value is ${u}
  _export: {u: "|~"}
"+for-0=u=4=%5B%3C%7B%7D%3E%5D":
  echo>: value is ${u}
  _export: {u: "[<{}>]"}
"+for-0=u=5=%23%3A":
  echo>: value is ${u}
  _export: {u: "#:"}
"+for-0=u=6=%2F%40":
  echo>: value is ${u}
  _export: {u: "/@"}
"+for-0=u=7=%2B%20":
  echo>: value is ${u}
  _export: {u: "+ "}
